lab_3_keywords_textrank/main.py

```diff
@@ -519,7 +595,8 @@ def update_vertex_score(self, vertex: int, incidental_vertices: list[int], score
             scores: dict[int, float]
                 scores of all vertices in the graph
         """
-        pass
+        self._scores[vertex] = 1 + self._damping_factor * (
```

Comment:
calculate `sum(scores[i] / self._graph.calculate_inout_score(i) for i in incidental_vertices) - 1` in separate line. It will be easer to understand